x-pack/plugins/serverless_observability/public/components/side_navigation/index.tsx

```diff
@@ -124,6 +124,29 @@ const navigationTree: NavigationTreeDefinition = {
             },
           ],
         },
+        {
+          id: 'metrics',
+          title: i18n.translate('xpack.serverlessObservability.nav.infrastructure', {
+            defaultMessage: 'Infrastructure',
+          }),
+          accordionProps: {
+            arrowProps: { css: { display: 'none' } },
+          },
+          children: [
+            {
+              link: 'metrics:inventory',
```

Comment:
Nit: maybe we can have similar link naming (as we have `metrics-hosts` we can have `metrics-inventory`) or we can have `metrics:inventory` and `metrics:hosts`, wdyt

Reply:
Good idea. Will change it.